js/views/playingScreen.js

```diff
@@ -69,6 +69,13 @@ var PlayingScreen = React.createClass({
     // for mobile, touch is handled in handleTouchEnd
   },
 
+  handleKeyPress: function(event){
+    // show control bar on tab key navegation
+    if(event.which === 9 || event.keyCode === 9){
```

Comment:
Small nitpick, can we add spaces before and after the parenthesis? Same for the function declaration above